src/Main.purs

```diff
@@ -108,9 +126,14 @@ main configJson = do
                 { cardanoMultiplatformLib
                 , walletInfoCtx
                 , logger
-                , contracts
+                , contractEvents: contractEmitter
```

Comment:
Cosmetics. Maybe let's change the name to `contractEmitter` in this record as well.